inst/assets/bibliography.bib

```diff
@@ -2763,3 +2763,29 @@ @article{clausen2025
   primaryClass={stat.ME},
   url={https://arxiv.org/abs/2402.05231},
 }
+
+@article{Jiang2025,
+  title={Multimedia: multimodal mediation analysis of microbiome data},
+  volume = {13},
+  url = {https://journals.asm.org/doi/10.1128/spectrum.01131-24?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed},
+  DOI = {10.1128/spectrum.01131-24},
+  number = {2},
+  journal = {Microbiology Spectrum},
+  publisher = {Epub},
+  author={Jiang, Hanying and Miao, Xinran and Thairu, Margaret W and Beebe, Mara and Grupe, Dan W and Davidson, Richard J and Handelsman, Jo and Sankaran, Kris},
+  year = {2025},
+  month = feb
+}
+
+@article{Jiang2025,
+  title={Multimedia: multimodal mediation analysis of microbiome data},
+  volume = {13},
+  url = {https://journals.asm.org/doi/10.1128/spectrum.01131-24?url_ver=Z39.88-2003&rfr_id=ori:rid:crossref.org&rfr_dat=cr_pub%20%200pubmed},
+  DOI = {10.1128/spectrum.01131-24},
+  number = {2},
+  journal = {Microbiology Spectrum},
+  publisher = {Epub},
+  author={Jiang, Hanying and Miao, Xinran and Thairu, Margaret W and Beebe, Mara and Grupe, Dan W and Davidson, Richard J and Handelsman, Jo and Sankaran, Kris},
+  year = {2025},
+  month = feb
+}
```

Comment:
This seems a duplicated entry. Can you remove the duplicate (or am I missing something)?